tests/mbed/enc/CMakeLists.txt

```diff
@@ -22,9 +22,27 @@ function(add_mbed_test_enclave NAME)
 
   set(MBEDTLS_TESTS_DIR "${PROJECT_SOURCE_DIR}/3rdparty/mbedtls/mbedtls/tests")
 
+  # For Win32 ELF enclave builds, use Perl from the Git Bash shell
+  if (WIN32)
```

Comment:
Instead of duplicating the bash finding logic, move the logic from:
https://github.com/Microsoft/openenclave/blob/master/3rdparty/CMakeLists.txt
upward to that ${BASH} is automatically available where needed.
E.g: https://github.com/Microsoft/openenclave/pull/1379/files